immutables/_map.pyi

```diff
@@ -0,0 +1,87 @@
+from typing import Any
+from typing import Generic
+from typing import Hashable
+from typing import Iterable
+from typing import Iterator
+from typing import Literal
+from typing import Mapping
+from typing import MutableMapping
+from typing import NoReturn
+from typing import Tuple
+from typing import Type
+from typing import TypeVar
+from typing import Union
+
+
+K = TypeVar('K', bound=Hashable)
+V = TypeVar('V', bound=Any)
+D = TypeVar('D', bound=Any)
+
+
+class BitmapNode: ...
+
+
+class MapKeys(Generic[K]):
```

Comment:
I guess mapping views should derive from corresponding `collections.abc`.
```suggestion
class MapKeys(Generic[K], collections.abc.KeysView):
```
in this case

Reply:
There's a corresponding generic type that we could use: https://docs.python.org/3/library/typing.html#typing.KeysView

Looking at [this table in the collections docs](https://docs.python.org/3/library/collections.abc.html#collections-abstract-base-classes) it looks like the views also need to implement `__contains__` to fulfill the interface?